oneview_redfish_toolkit/tests/blueprints/test_computer_system.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import json
+import unittest
+from unittest import mock
+
+# 3rd party libs
+from flask import Flask
+from flask_api import status
+from hpOneView.exceptions import HPOneViewException
+from oneview_redfish_toolkit import util
+
+# Module libs
+from oneview_redfish_toolkit.blueprints.computer_system import computer_system
+
+
+class TestComputerSystem(unittest.TestCase):
+    """Tests for ComputerSystem blueprint
+
+        Tests:
+            - server hardware not found
+            - server hardware types not found
+            - oneview exception server hardware
+            - oneview exception server hardware type
+            - oneview unexpected exception
+            - know computer system
+    """
+
+    @mock.patch.object(util, 'OneViewClient')
+    def setUp(self, ov_mock):
+        """Tests preparation"""
+
+        # Load config on util
+        util.load_config('oneview_redfish_toolkit/redfish.ini')
+
+        # creates a test client
+        self.app = Flask(__name__)
+
+        self.app.register_blueprint(computer_system)
+
+        self.app = self.app.test_client()
+
+        # propagate the exceptions to the test client
+        self.app.testing = True
+
+    @mock.patch.object(util, 'get_oneview_client')
+    def test_get_computer_system_sh_not_found(self, mock_get_ov_client):
```

Comment:
You should test if the response.date is equal to the response of not_found(error) method in computer system error handler.